Providers/nxOMSAutomationWorker/automationworker/worker/omsutil.py

```diff
@@ -79,6 +79,16 @@ def initialize():
     else:
         print "Successfully set owners of certificate folder."
 
+    proxy_paths = ["/etc/opt/microsoft/omsagent/conf/proxy.conf", "/etc/opt/microsoft/omsagent/proxy.conf"]
```

Comment:
The path `/etc/opt/microsoft/omsagent/proxy.conf` should be used as the default path, so I think it should be moved to the first position in the array for clarity. FYI.